### PR TITLE
Update RTD site to redirect to RAPIDS docs

### PR DESCRIPTION
**Pull request by charlesbluca**
Taking @jakirkham's [suggestion](https://github.com/rapidsai/dask-cuda/issues/579#issuecomment-844333374) for handling RTD redirects to an external site.

### CODE REVIEW

**Review by pentschev — APPROVED** [review]
LGTM, thanks @charlesbluca .

@jakirkham could you also take a look?

File: rtd/conf.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import shutil
+
+# -- Project information -----------------------------------------------------
+
+project = "dask-cuda"
+copyright = "2020, NVIDIA"
```
Comment (jakirkham) [inline rtd/conf.py:21]:
```suggestion
copyright = "2020-21, NVIDIA"
```

**Comment by jakirkham** [comment]
Yep looks good. Had one suggestion above. Thanks for working on this Charles 😄

**Comment by charlesbluca** [comment]
Thanks @jakirkham! Based on that suggestion, we should also probably update the copyright of the main docs as well - would you be okay with using `datetime` to update this automatically? Something like

```python
copyright = "2020-%s, NVIDIA" % datetime.datetime.now().year
```

**Comment by jakirkham** [comment]
Agreed that is better :)

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
@gpucibot merge

**Comment by jakirkham** [comment]
Manually started a build on RTD. Passed without issues. The redirect seems to be working 🎉

**Comment by jakirkham** [comment]
Thanks Charles! 😄

**Pull request merged**